compiler/src/iree/compiler/Codegen/Common/LinkTuningSpecsPass.cpp

```diff
@@ -83,7 +83,9 @@ emitLinkedTuningSpec(ModuleOp module, ArrayRef<NamedSequenceOp> specsToLink) {
       0, hasConsumedSequences ? kArgConsumedAttrName : kArgReadOnlyAttrName,
       builder.getUnitAttr());
   newSpec->setAttr(kTuningSpecEntrypointAttrName, builder.getUnitAttr());
-  module->setAttr(kTuningSpecDefaultEntrypointAttrName, builder.getUnitAttr());
+  // TODO: re-enable default attribute as below once new linking lands.
```

Comment:
```suggestion
  // TODO: Re-enable default attribute as below once new linking lands.
```
nit: Use proper capitalization per https://llvm.org/docs/CodingStandards.html#commenting.